build/antrea/base/Dockerfile.ubi

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu:20.04 as cni-binaries
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends wget ca-certificates
+
+# Leading dot is required for the tar command below
+ENV CNI_PLUGINS="./host-local ./loopback ./portmap ./bandwidth"
+
+RUN mkdir -p /opt/cni/bin && \
+    wget -q -O - https://github.com/containernetworking/plugins/releases/download/v0.8.7/cni-plugins-linux-amd64-v0.8.7.tgz | tar xz -C /opt/cni/bin $CNI_PLUGINS
+
+
+FROM antrea/ovs-ubi:2.14.0
```

Comment:
antrea upstream has updated ovs to 2.14.2, please refer to https://github.com/antrea-io/antrea/pull/2451

Reply:
> antrea upstream has updated ovs to 2.14.2, please refer to [antrea-io/antrea#2451](https://github.com/antrea-io/antrea/pull/2451)

This patch is for generating Antrea 0.13.1, should we use ovs 2.14.2 in Antrea 0.13.1?

Reply:
ok. This patch has backport to antrea 0.13-release, and antrea upstream introduces this patch because there are 3 CVEs, by the way, antrea already has 0.13.4, do you still need 0.13.1?

Reply:
> ok. This patch has backport to antrea 0.13-release, and antrea upstream introduces this patch because there are 3 CVEs, by the way, antrea already has 0.13.4, do you still need 0.13.1?

Yes, after discussed with danting, we decided to use 0.13.1 currently. For future, we'll find a way to sync the manifests and build UBI image when Antrea releases new version.